backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -35,6 +35,9 @@ package object backend {
                                   key: BackendJobDescriptorKey,
                                   inputs: Map[LocallyQualifiedName, WdlValue]) {
     val call = key.call
+
+    /** The abort functionality potentially becomes available after the job descriptor is created */
+    var abortFunction: Option[() => Unit] = None
```

Comment:
So the `receive` method for `JesAsyncBackendJobExecutionActor` is in `AsyncBackendJobExecutionActor`, so if we were to send the abort message there, it couldn't do JES specific abort functionality because `AsyncBackendJobExecutionActor` is in the backend package.

Also, the real reason for the `var` is to know record the RunID when it becomes available to know if I can even abort in the first place... If I don't have this `var`, I'd have to go to the metadata service, I guess?